tests/unit/hive_metastore/test_catalog_schema.py

```diff
@@ -70,6 +70,14 @@ def prepare_test(ws, backend: MockBackend | None = None) -> CatalogSchema:
                     'src_table': 'abfss://container@msft/path/dest3',
                     'workspace_name': 'workspace',
                 },
+                {
+                    'catalog_name': 'catalog4',
+                    'dst_schema': 'schema4',
+                    'dst_table': 'table1',
+                    'src_schema': 'schema1',
+                    'src_table': 'abfss://container@msft/path/dest3',
```

Comment:
```suggestion
                    'src_table': 'abfss://container@msft/path/dest4',
```

Reply:
@JCZuurmond Thanks for reviewing. Updated this.